### DISCUSSION TITLE
神策埋点怎么测试啊？有没有简单的栗子？

### DISCUSSION THREAD

**Discussion by murph-zh** [Pactortester/garbevents#21] [Q&A] [answered]
神策埋点怎么测试啊？有没有简单的栗子？

**Comment by Pactortester** [comment, marked as answer]
- 使用说明看这个链接：https://github.com/Pactortester/garbevents#%E7%A5%9E%E7%AD%96%E6%95%B0%E6%8D%AE-demo-1
- 如下图所示
![image](https://user-images.githubusercontent.com/29191106/173028664-3798abc8-499b-4c75-a018-2364423751e6.png)
